src/test/regress/expected/foreign_tables_mx_0.out

```diff
@@ -0,0 +1,407 @@
+\set VERBOSITY terse
```

Comment:
Should we add the test to `pg14.sql` test file to avoid this additional test output?

It is not nice to have additional test outputs.

I think it is fine to do `citus_add_node` for the coordinator on pg14.sql.